### PR TITLE
Buffs medbots massively.

### PR DESCRIPTION
**Pull request by Putnam3145**
## About The Pull Request

Medbots are now better at noticing your minor burns.

## Why It's Good For The Game

What is the meaning of "good"?

## Changelog
:cl:
balance: Medbots are finally worth making.
/:cl:

### CODE REVIEW

**Review by deathride58 — APPROVED** [review]
should i mark this as eligible for speedmerge
i feel this is the single most important change to ever be conceived in the history of ss13

**Comment by Linzolle** [comment]
thanks I hate it

**Comment by DeltaFire15** [comment]
thanks I hate it.

**Comment by MrJWhit** [comment]
Thanks I love it.

**Comment by silicons** [comment]
OwO

**Comment by IronEleven** [comment]
This buff is long overdue.

File: code/modules/mob/living/simple_animal/bot/medbot.dm [hunk 1]
```diff
@@ -264,6 +264,9 @@
 		if((last_newpatient_speak + 300) < world.time) //Don't spam these messages!
 			var/list/messagevoice = list("Hey, [H.name]! Hold on, I'm coming." = 'sound/voice/medbot/coming.ogg',"Wait [H.name]! I want to help!" = 'sound/voice/medbot/help.ogg',"[H.name], you appear to be injured!" = 'sound/voice/medbot/injured.ogg')
 			var/message = pick(messagevoice)
+			if(H.getFireLoss() && H.getFireLoss() < 20 && prob(1))
```
Comment (Ghommie) [inline code/modules/mob/living/simple_animal/bot/medbot.dm]:
```suggestion
			if(prob(1) && ISINRANGE_EX(H.getFireLoss(), 0, 20))
```

**Pull request merged**